### PR TITLE
Provide some more granular metrics for polkadot_pvf_execution_time

### PR DESCRIPTION
**Pull request by m-cat**
Previously the metrics were mostly clustered at under 100ms which wasn't very useful. Also, there were some jobs that finished between 6s and +Inf which likewise didn't tell us much.

### Before

Screenshot 2022-11-25 at 07 30 43

### After

How do I get an after pic?

### CODE REVIEW

File: node/core/pvf/src/metrics.rs [hunk 1]
```diff
@@ -192,6 +195,10 @@ impl metrics::Metrics for Metrics {
 						4.0,
 						5.0,
 						6.0,
+						7.0,
+						8.0,
+						9.0,
+						10.0,
```
Comment (ordian) [inline node/core/pvf/src/metrics.rs:201]: > This is synchronized with `APPROVAL_EXECUTION_TIMEOUT`

which is currently 12s, so should be the upper bracket?

**Review by ordian — APPROVED** [review]

**Review by sandreim — APPROVED** [review]
To get an after pick, you would need a build with these merged to be deployed on testnet. Or eventually it wil make it in the release that is deployed on our Kusama monitoring nodes and you can see it here: https://grafana.parity-mgmt.parity.io/goto/hvVSDlOVz?orgId=1

**Comment by sandreim** [comment]
The reason why PVF takes mostly under 100ms is that the parachain blocks are mostly empty.

**Review by s0me0ne-unkn0wn — APPROVED** [review]

**Comment by m-cat** [comment]
> The reason why PVF takes mostly under 100ms is that the parachain blocks are mostly empty.

Ah got it. That does make these metrics less useful, unfortunately. Is there a reason we don't try to simulate the production chains more closely?

I tried looking at prod metrics by selecting "polkadot" as the chain, but it said "No Data". I didn't see a Kusama option, so I selected "westend2" but all the times there were under 100ms as well.

**Comment by burdges** [comment]
We want some parachains which we configure to waste CPU cycles, blockspace, storage r/w, etc.  see https://github.com/paritytech/substrate/issues/12772

**Review by bredamatt — APPROVED** [review]

**Comment by bredamatt** [comment]
Good idea @m-cat

**Pull request merged**